src/App.css

```diff
@@ -0,0 +1,11 @@
+footer {
+  position: absolute;
+  text-align: center;
+  bottom: 0;
+  width: 100%;
+  height: 2.5rem;
+}
+
+footer p {
+  color: #ccc;
+}
```

Comment:
So, if you call this file `App.css` then you kind of scope the CSS that this file contains to only be used in the `App` component. And it does not, right?  So you have 2 choices in my opinion. You either call it `index.css` and import it in `index.js` (as you do below), or you call it `Footer.css` and import it in the `Footer` component. 

I would suggest the first variant but you need to form your own opinion on this though.